README.md

```diff
@@ -9,9 +9,10 @@ These are things you will need to acquire in order to run the CFN Template in AW
 ## Slack
 You will need to generate a slack token for hubot.
 
-1. Go to https://$yourteam.slack.com/services/new/hubot and create a new hubot integration
-2. Choose a bot username, this is what you will call to the bot in rooms as (e.g.; @hubot help)
-3. Find the API Token section (the token will start with `xoxb-`)
+1. [Hubot Slack App](https://slack.com/apps/A0F7XDU93-hubot)
```

Comment:
why have both 1 and 2? Step two takes you directly to the create username for a new hubot. 

We need to keep only one and if it's step 1, then we need to add additional steps to cover the screens.